consensus/Cargo.toml

```diff
@@ -19,13 +19,13 @@ thiserror = "1.0.35"
 tokio = { version = "1.20.1", features = ["sync"] }
 tracing = "0.1.36"
 
-config = { path = "../config" }
+config = { path = "../config", package = "narwhal-config" }
 fastcrypto = "0.1.2"
-crypto = { path = "../crypto" }
-storage = { path = "../storage" }
-dag = { path = "../dag" }
+crypto = { path = "../crypto", package = "narwhal-crypto" }
+storage = { path = "../storage", package = "narwhal-storage" }
+dag = { path = "../dag", package = "narwhal-dag" }
 prometheus = "0.13.2"
-types = { path = "../types" }
+types = { path = "../types", package = "narwhal-types" }
```

Comment:
my preference would be to probably import the crates with the name being "narwhal-*" instead of renaming them to "*" as we're doing in this case but i suppose that would result in quite a bit more churn atm

Reply:
Yeah the churn will be larger if we do this additional renaming. It can be a separate change in future.